index.html

```diff
@@ -21,10 +21,16 @@
           <nav class="hamburger-icon">
             <img src="photo/Icon - Menu.png" alt="hamburger-icon">
           </nav>
+          <UL class="header-links">
+            <li class="header-link">Portfolio</li>
+            <li class="header-link">About</li>
+            <li class="header-link">Contact</li>
+          </UL>
       </header>
       <div class="main">
         <h1>
-           I<span>&#39;</span>m Khaled Glad to see you! 
+           I<span>&#39;</span>m Khaled 
+           <br> Glad to see you! 
         </h1>
         <p class="Main-Text">
           I<span>&#39;</span>m a software developer! I can help you build a
```

Comment:
- [ ] Good job in this section 👏 . However, it seems your background image has an overlap that clusters the background. the initial figma design has just one, which makes it simple yet professional and with a great user experience. Kindly adjust this based on the comparison below, to complement your great work so far 👍 

## Your view :
![image](https://user-images.githubusercontent.com/50990925/222937180-a3cdf108-ddff-409a-8714-bb0117df50f3.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/222937191-34f09c58-71ea-44a3-a144-1f9c22d367a4.png)